mantidproject/mantid
# TF Asymmetry mode bug fixes

**Description of work.**
Some bug fixes for the muon TFAsymmetry mode. 

**Report to:** James. 

**To test:**


Open Muon Analysis
Go to the settings tab and make sure multiple fitting is ticked
Go back to the home tab
Load MUSR 62260
Go to the Data analysis tab
Add a GaussOsc function, set frequency and phase to global
Turn on the TF Asymmetry mode (its in the settings table of the data analysis tab)
The function would have changed, but if you scroll down you will find the GaussOcs with the same values and globals as you set earlier.
Do a fit
At the end a chi squared will appear in the grey bar at above the fit menu (this is new).
Turn of the TF Asymmetry mode (either untick the box or leave and come back to the tab)
The values should be preserved and the gloabls (this bit is new).

Fixes #23138. 




---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
